geocropper/utils.py

```diff
@@ -1837,3 +1837,117 @@ def move_crops_containing_locations(csv_path, source_dir, target_dir):
                             target_dir.mkdir(parents=True, exist_ok=True)
                             shutil.move(str(crop.absolute()), str(target_dir.absolute()))
                             break                 
+
+
+def get_unique_lat_lon(root_dir, speckle):
+    """Loops through every folder of the specified speckle variant in rootdir and returns 
+    a set with unique latitude and longitude positions.
+
+    Parameters
+    ----------
+    root_dir: Path
+        Path of trimmed crops with various speckle variants and recording times.
+    speckle: String
+        Set desired speckle variant (options available in geocropper.stack_trimmed_images) 
+    """
+    
+    # Set is an unordered list which allows no duplicated entries
+    lat_lon_set = set()
+
+    print(f"Reading unique positions for {speckle}...")
+
+    for folder in root_dir.glob(f"*{speckle}*"):
+        if folder.is_dir() and folder.name.split("_")[0] != "stacked":
+            for crop in folder.glob("*"):
+                if crop.is_dir() and crop.name != "0_combined-preview":

```

Comment:
I sometimes created multiple combined-preview folders and renamed them. So this would be better:
` and crop.name.startswith("0_"):`

Reply:
I implemented this suggestion with the exception that it should NOT start with "0_":
`and not crop.name.startswith("0_")`

Reply:
Yeah my fault I meant "and not". I just wanted to change "0_combined-preview" to "0_" since the name may be different sometimes, but there is no crop with ID 0